src/resource/note/dto/note.dto.ts

```diff
@@ -1,28 +1,51 @@
-import { IsOptional, IsString } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { IsOptional, IsString } from 'class-validator';
 
```

Comment:
Remove constructors from dto (u know how) and remove expose, and all mappings to entity in dto, dto in dto should be only allowed.